.circleci/config.yml

```diff
@@ -182,7 +182,7 @@ jobs:
       - image: continuumio/miniconda3
     steps:
       - checkout
-      - check_changes
+      # - check_changes
```

Comment:
that was there so that Circle should run the build test for us to test the actual build, otherwise on a normal commit trigger it'd skip it (as it should, I put it back in, we've done the necessary testing)